shader/lighting/lighting_uber.glsl

```diff
@@ -31,6 +31,13 @@ void main()
 
 layout(location = 0) out vec4 out_color;
 
+float lin(float d){
```

Comment:
What is this function?

Reply:
Removed. In use when I did not know about existing helper functions.